### PR TITLE
[new release] blink (0.0.1)

### PR DESCRIPTION
**Pull request by leostera**
A short synopsis

- Project page: <a href="https://github.com/leostera/blink">https://github.com/leostera/blink</a>

## 0.0.1

First release of Blink, including:

* Create and reuse connectionsc
* Stream requests into connection messages
* Support for clear TCP and SSL transports

### CODE REVIEW

**Comment by leostera** [comment]
Depends on https://github.com/ocaml/opam-repository/pull/24991

**Comment by arbipher** [comment]
The failed case is on missing of package `httpaf`.
`blink/dune` requires `httpaf` while `dune-project` doesn't specify it. I wonder should `httpaf` be listed there?

File: packages/blink/blink.0.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+license: "MIT"
+tags: ["http" "client" "http client" "riot" "multicore"]
+homepage: "https://github.com/leostera/blink"
+bug-reports: "https://github.com/leostera/blink/issues"
+depends: [
+  "dune" {>= "3.11"}
+  "ocaml" {>= "5.1"}
+  "riot" {>= "0.0.7"}
+  "http" {>= "6.0.0~alpha2"}
+  "castore" {>= "0.0.1"}
```
Comment (mseri) [inline packages/blink/blink.0.0.1/opam:15]:
```suggestion
  "httpaf"
  "castore" {>= "0.0.1"}
```
Looking at the sources it ja using it, should be added as dependency

**Comment by mseri** [comment]
Tests here cannot work since they try to access the network:
```
#=== ERROR while compiling blink.0.0.1 ========================================#
# context              2.2.0~alpha4~dev | linux/x86_64 | ocaml-base-compiler.5.1.1 | pinned(https://github.com/leostera/blink/releases/download/0.0.1/blink-0.0.1.tbz)
# path                 ~/.opam/5.1/.opam-switch/build/blink.0.0.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p blink -j 255 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/blink-7-702778.env
# output-file          ~/.opam/log/blink-7-702778.out
### output ###
# File "test/dune", line 2, characters 7-25:
# 2 |  (name get_ocaml_org_test)
#            ^^^^^^^^^^^^^^^^^^
# (cd _build/default/test && ./get_ocaml_org_test.exe)
# 2023-12-29 11:38:05.91166Z ERROR [thread=0] Process <0.1.0> died with unhandled exception Invalid_argument("result is Error _"):
# Raised at Stdlib.invalid_arg in file "stdlib.ml", line 30, characters 20-45
# Called from Runtime__Import._spawn.(fun) in file "riot/runtime/import.ml", line 92, characters 10-15
# 
# 
```

File: packages/blink/blink.0.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+homepage: "https://github.com/leostera/blink"
+bug-reports: "https://github.com/leostera/blink/issues"
+depends: [
+  "dune" {>= "3.11"}
+  "ocaml" {>= "5.1"}
+  "riot" {>= "0.0.7"}
+  "http" {>= "6.0.0~alpha2"}
+  "httpaf" {>= "0.7.0"}
+  "castore" {>= "0.0.1"}
+  "cohttp" {>= "5.3.1"}
```
Comment (mseri) [inline packages/blink/blink.0.0.1/opam:17]:
```suggestion
```
Looking at the sources you don’t seem to need this

**Comment by leostera** [comment]
Hi folks! Yes :) I missed swapping out `cohttp` for `httpaf` here. I think I'll park this one until we release Riot 0.0.8 where the SSL support will be one layer below.

Thanks for the review tho, I've amended the dune-project to fix these issues 🙏🏼 closing for now!

**Pull request closed without merging**